documentation/schema.json

```diff
@@ -1063,6 +1063,15 @@
           ],
           "description": "Enables or disables the exceptions feature. By default, this feature is enabled if in-process features are enabled.",
           "default": true
+        },
+        "TopLevelLimit": {
+          "type": [
+            "integer",
+            "null"
+          ],
+          "description": "The number of top-level exceptions to keep in the cache before automatically removing older exceptions. A top-level is exception is one that is not an inner exception of another exception.",
+          "format": "int32",
+          "default": 20
```

Comment:
Apologies if this was already covered, but do we filter certain exception types by default? For example, I could see OperationCancelledException starving out every other exception type if we only have 20 exceptions.

Reply:
No, we do not have any filter whatsoever at this time.